src/Sulu/Bundle/AdminBundle/Resources/js/containers/Login/ResetPasswordForm.js

```diff
@@ -25,7 +33,7 @@ class ResetPasswordForm extends React.Component<Props> {
 
     @observable inputRef: ?ElementRef<*>;
 
-    @observable error: boolean;
+    @observable error: ?string = null;
```

Comment:
would call this `errorMessage` and directly set the translated message that is displayed